src/lib/orion_websocket/parser.cpp

```diff
@@ -30,10 +30,19 @@
 #include "rapidjson/writer.h"
 
 #include "logMsg/logMsg.h"
+#include "rest/HttpHeaders.h"
 
-void ws_parser_parse(std::string &url, std::string &verb, std::string &payload, const char *msg)
+
+#include <map>
+
+
+void ws_parser_parse(const char *msg,
+                     std::string &url, std::string &verb,
+                     std::string &payload, HttpHeaders &head)
 {
```

Comment:
Remember, for all old code without standards I have a task in this sprint to fix that ;)